src/resolvers/admin.ts

```diff
@@ -23,4 +24,20 @@ export class AdminResolver {
     await killSwitch.save()
     return { isKilled }
   }
+
+  @Mutation(() => GeneratedSignature)
+  async generateSignature(@Arg('bytes', () => String) bytes: string) {
+    if (!config.appPrivateKey) {
+      throw new Error('No PRIVATE_KEY provided to generate signature')
+    }
+
+    if (!cryptoIsReady()) {
+      throw new Error('@polkadot/util-crypto is not ready')
+    }
+
+    const keypair = sr25519PairFromSeed(config.appPrivateKey)
```

Comment:
This should be done on startup or as part of config, not on request. This way if the operator provided some kind of wrong input, the app will let them know right away